public/pages/Aliases/containers/DeleteAliasModal/DeleteAliasModal.tsx

```diff
@@ -63,7 +75,7 @@ export default function DeleteAliasModal(props: DeleteAliasModalProps) {
     return null;
   }
 
-  const hasSystemIndex = props.selectedItems.some((index) => filterByMinimatch(index, SYSTEM_ALIAS));
+  const hasSystemIndex = props.selectedItems.some((index) => filterByMinimatch(index.alias, SYSTEM_ALIAS));
```

Comment:
Is this a different bug?

Reply:
Nope, the selectedItems in the props changed, it was a string[] but now it's a IAlias[].